pkg/controllermanager/controller/seed/seed_extension_check_control_test.go

```diff
@@ -79,7 +82,8 @@ var _ = Describe("ExtensionCheckReconciler", func() {
 	AfterEach(func() {
 		if err := c.Get(ctx, request.NamespacedName, seed); !apierrors.IsNotFound(err) {
 			Expect(err).NotTo(HaveOccurred())
-			Expect(seed.Status.Conditions).To(ConsistOf(expectedCondition))
+			Expect(seed.Status.Conditions).NotTo(BeZero())
```

Comment:
```suggestion
			Expect(seed.Status.Conditions).NotTo(BeEmpty())
```

makes more sense / can be read easier